How-To-Host-Feed.md

```diff
@@ -345,10 +345,10 @@ See https://inedo.com/support/documentation/proget/installation/installation-gui
 #### ProGet Enterprise High Availabilty
 
 * ProGet Server (Each): Windows Server with 4+ cores (CPU), 8GB+ RAM, and probably at least 50GB of free space (as the artifacts are stored on shared storage). This is similar to ProGet section above.
-* Shared Storage: Appliance-based (such as NAS), or software-based (such as SMB). This will be shared across all Servers, so they will need to be able to read and write to it at high speed. Recommend appliance-based (NAS) with 3 times the total size of what you will store (and a backup) - 2-5TB+ space.
-* Database: SQL Server with Database Clustering Recommended. Assume high specs for SQL Server, pretty much the highest specs you can configure on a Windows machine. 16+ cores (CPU), 64GB+ RAM, 5TB+, etc
-* Load Balancer: Assume powerful - this will be software-based (such as nginx or HAProxy) or an appliance (such as F5 or Citrix).
-* May or may not support cross-datacenter replication.
+* Shared Storage: Shared Storage: Compatible with any type of common storage that web and indexing nodes can access, whether software-based (SAMBA share, Windows Server Storage Spaces, etc) or appliance-based (dedicated NAS). The only requirement is that the storage is readable and writeable by all nodes. Recommend appliance-based (NAS) with 3 times the total size of what you will store (and a backup) - 2–5TB+ space.
```

Comment:
We don't spell out SMB anywhere in our documentation - let's keep that consistency.